doc/installation.rst

```diff
@@ -95,8 +97,8 @@ Fetch the source code::
 
 .. _seqrepo_install:
 
-Installing SeqRepo (recommended)
-@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+Installing SeqRepo (recommended, **required for Windows**)
```

Comment:
@aaronslaff: Other way around: seqrepo doesn't work on windows (at all), except in a docker image.